logstreams/pom.xml

```diff
@@ -73,6 +73,11 @@
       <artifactId>atomix-storage</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.zeebe</groupId>
+      <artifactId>zeebe-snapshots</artifactId>
```

Comment:
Do we still need to snapshot classes here?

Reply:
Only used by log deletetion test, can be moved to broker.